Dockerfile

```diff
@@ -115,8 +111,9 @@ RUN DEBIAN_FRONTEND=noninteractive \
       postgresql-9.6 \
       postgresql-client-9.6 \
       postgresql-contrib-9.6 \
-      google-chrome-stable \
+      git-restore-mtime \
       nodejs \
+      libgbm1 \
```

Comment:
So, we previously downloaded chrome because Playwright needed it, but we can get away with only libgbm1 somehow?

Reply:
No, we previously downloaded chrome because testcafe needed it, but now playwright bundles it itself, requiring just the dependencies on the system.